tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -912,7 +912,7 @@ def test_import_tensorflow_with_fluent_autolog_enables_tf_autologging():
 
     # NB: For backwards compatibility, fluent autologging enables TensorFlow and
     # Keras autologging upon tensorflow import in TensorFlow 2.5.1
-    if Version(tf.__version__) != Version("2.5.1"):
+    if Version(tf.__version__) >= Version("2.6"):
```

Comment:
Why do we also want to disable `Version(tf.__version__) < Version("2.5.1")`? it looks like only 2.5.1 and 2.5.2 are causing failures.

Reply:
@dbczumar 
I think previous code logic here is wrong. Only >= 2.6 version will redirect keras autologging to tensorflow autologging.

Reply:
CC @liangz1 

For more context, you can read this PR:
https://github.com/mlflow/mlflow/pull/4766

Simple explanation:
In Tensorflow >= 2.6, the Tensorflow embedded keras will be linked to the solely installed keras (in contrast, previous Tensorflow embedded keras was a different module with the solely installed keras)

The change in Tensorflow >= 2.6, cause the mlflow.tensorflow.autolog (including patching on tf and embeded tf.keras) and mlflow.keras.autolog (patching on keras ) conflicts. So, as a workaround, for TF >= 2.6, we disable mlflow.keras.autolog, but instead, we only trigger mlflow.tensorflow.autolog